brayns/common/PropertyMap.h

```diff
@@ -69,14 +69,34 @@ class PropertyMap
         }
 
         template <typename T>
-        Property(const std::string& name_, const std::string& title_,
-                 const T& value, const T& min_, const T& max_)
+        Property(const std::string& name_, const std::string& label_,
+                 const T& value, const std::pair<T, T>& limits)
             : name(name_)
-            , title(title_)
+            , label(label_)
             , type(_getType<T>())
             , _data(value)
-            , _min(min_)
-            , _max(max_)
+            , _min(limits.first)
```

Comment:
Should it not be just `limit` and `limit.min`/`limit.max`? `limits` would implies there is a set of limit pairs.